content/news/171-hard-fork-july-2018.md

```diff
@@ -0,0 +1,31 @@
+---
+author: alex-grintsvayg
+title: 'Hard Fork on July 9th, 2018'
```

Comment:
I would propose we have some kind of naming convention for hard and soft forks so that we can refer to them canonically.

Right now I would call this something like `hardfork_2018-07-09`, but there may be a better pattern than this. Worth looking at what others do.

Reply:
I have not been able to find any standard for this. I think its unlikely that we'll do more than one fork in a month, so just year and month should be enough. I propose we call this one `HF1807`.